lambdas/db-migration/src/migrations/20201014105024_create_rules_table.ts

```diff
@@ -0,0 +1,66 @@
+import * as Knex from 'knex';
+
+export const up = async (knex: Knex): Promise<void> =>
+  knex.schema.createTable('rules', (table) => {
+    table
+      .increments('cumulusId')
+      .primary();
+    table
+      .text('name')
+      .comment('Rule name')
+      .notNullable();
+    table
+      .text('workflow')
+      .comment('Workflow name to invoke for this rule')
+      .notNullable();
+    table
+      .integer('collectionCumulusId')
+      .references('cumulusId')
+      .inTable('collections')
+      .notNullable();
+    table
+      .integer('providerCumulusId')
+      .references('cumulusId')
+      .inTable('providers')
+      .notNullable();
+    table
```

Comment:
It's probably worth calling out that this used to be nested under the `rule` object on the Dynamo schema, so `rule.type`. But given that part of the point of going to RDS is to facilitate easier querying on any record field, it seemed worth pulling all the `rule.*` properties to the top level (`type`, `value`, `arn`, `logEventArn`)

Reply:
I think that's fine. We'll need to keep it in mind when adding rule parallel writes to transform, but that'll shake out.